Bottom: 36145a4ced59be217f2a3b7bebbbd8ee76f523bb
Top:    3cd650b75c3827c4cb50b12db57205f72d1345dd
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-21 10:47:50 +0300

Refresh of ctl-verbs.patch

---

diff --git a/ib-sock-ctl.c b/ib-sock-ctl.c
index 16b76ce..f334383 100644
--- a/ib-sock-ctl.c
+++ b/ib-sock-ctl.c
@@ -47,8 +47,9 @@ int ib_sock_ctl_post(struct IB_SOCK *sock, struct ib_sock_ctl *msg)
 	 */
 	struct ib_recv_wr wr;
 	struct ib_recv_wr *bad_wr;
+	int ret;
 
-	ret = ib_post_recv(rx->qp, &wr, &bad_wr);
+	ret = ib_post_recv(sock->is_qp, &wr, &bad_wr);
 
 	return 0;
 }
